languages/elixir/reference/README.md

```diff
@@ -257,24 +257,25 @@ TODO: Flesh out how the above general concepts apply to concept exercises in the
-| `anonymous-functions`            | Intro to anonymous functions, functions as data                                                |
-| `atoms`                          | Intro to Elixir atom type.                                                                     |
-| `basics`                         | Introduction to functions, modules, variables, returning values, integers, invoking functions. |
-| `bit manipulation`               | Introduction to bit manipulation using the Bitwise module functions                            |
-| `booleans`                       | Introduction to the boolean type and strict boolean operators -- and/2, or/2, not/1            |
-| `default-arguments`              | Introduction to default arguments in named functions, function headers                         |
-| `closures`                       | How to implement closures in Elixir                                                            |
-| `conditionals`                   | Intro to Elixir `cond/1` function.                                                             |
-| `floating-point-numbers`         | How to use floating point numbers to represent real numbers                                    |
-| `guards`                         | What guards are, how to use guards in function heads                                           |
-| `lists`                          | Introduction to the lists type basic list functions -- hd/1, tl/1, length/1, in/2              |
-| `maps`                           | Introduction to the map data type                                                              |
-| `module-attributes-as-constants` | Introduction to using module attributes as constants                                           |
-| `multiple-clause-functions`      | Named function can be overloaded and each attempted to invoke until one succeeds               |
-| `pattern-matching`               | Basic knowledge of pattern matching using `=/2` and on function parameters                     |
-| `recursion`                      | How to write basic recursive functions                                                         |
-| `string-literals`                | Introduction to strings in code enclosed by double quotes.                                     |
-| `strings`                        | How to do string processing, concatenation, interpolation, and multiline strings.              |
-| `tuples`                         | Introduction to the tuple data type                                                            |
+| concept                          | interpretation                                                                                          |
+| -------------------------------- | ------------------------------------------------------------------------------------------------------- |
+| `access-behavior`                | Brief overview of behaviors, using access behavior with maps, knowing it can be used with keyword lists |
```

Comment:
```suggestion
| `access-behaviour`               | Brief overview of behaviours, using access behaviour with maps, knowing it can be used with keyword lists |
```

Reply:
I am committing this suggestion, it may cause prettier to fail, will amend with a follow up commit